src/main/java/com/ee/employeesportal/repositories/JpaEmployeeRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.ee.employeesportal.repositories;
+
+import com.ee.employeesportal.model.Employee;
+import org.springframework.data.jpa.repository.JpaRepository;
+
+import java.util.List;
+
+public interface JpaEmployeeRepository extends JpaRepository<Employee,Integer> {
```

Comment:
`JpaRepository<Employee,Integer>` should be `JpaRepository<Employee,Long>` because the primary key in `Employee` entity is `Long` type.